contrib/presets/colors/protanopia-deuteranopia.conf

```diff
@@ -0,0 +1,105 @@
+# Preset for Red-green dichromacy color blindness.
```

Comment:
At least for me, this didn't help much. Removing red entirely means you have less color options, meaning each color is less different. Which is the opposite of what I need.

Red and green look the same, but only if they are dark. The brighter they are, the easier they are to tell apart. A nice light green like `lp_terminal_format 84` and a bright red like `lp_terminal_format 196` are *very* easy to tell apart. It also helps that the relative intensities are not the same.

I struggle more with differences between light yellows and light greens, so the `LP_COLORMAP` here is very difficult to use. I think a spectrum of light to dark (as well as colors shifting, probably from green to blue to red?) would be much easier to use, say:
```bash
for num in 84 154 225 33 21 196 124 ; do
  lp_terminal_format $num
  printf '%s\n' "$lp_terminal_format foobar$(tput sgr0)"
done
```
Each value there is very clearly different to me.

But those are just my thoughts. And I have never been officially diagnosed, so I don't even know if I have Protanopia or Deuteranopia.

Reply:
I've read some explanations since the PR and it seems indeed that the mild conditions are generally treated differently than the severe ones. So I guess we should probably have presets for both cases, with either color removal (severe case), or contrast increase (mild case).

But then I am wondering: can we really have more contrast than the 16 colors palette? Especially considering that most terminal emulator allow for fine-tuning those colors. In which case, the default theme may very well be the best we can do for mild cases.

Reply:
That's fair. It is not an easy problem to solve.

It's also possible that we can't even come up with solutions that work for everyone or even multiple people.

Reply:
I renamed the presets, so that the underlying claims are less bold (and maybe easier to understand, also).